lib/exe/mediamanager.php

```diff
@@ -2,36 +2,36 @@
 
 use dokuwiki\Extension\Event;
 
-    if(!defined('DOKU_INC')) define('DOKU_INC',__DIR__.'/../../');
-    define('DOKU_MEDIAMANAGER',1);
+    if(!defined('DOKU_INC')) define('DOKU_INC', __DIR__.'/../../');
+    define('DOKU_MEDIAMANAGER', 1);
 
     // for multi uploader:
-    @ini_set('session.use_only_cookies',0);
+    @ini_set('session.use_only_cookies', 0);
 
     require_once(DOKU_INC.'inc/init.php');
 
     global $INPUT;
     global $lang;
     global $conf;
     // handle passed message
-    if($INPUT->str('msg1')) msg(hsc($INPUT->str('msg1')),1);
-    if($INPUT->str('err')) msg(hsc($INPUT->str('err')),-1);
+    if($INPUT->str('msg1')) msg(hsc($INPUT->str('msg1')), 1);
+    if($INPUT->str('err')) msg(hsc($INPUT->str('err')), -1);
 
     global $DEL;
     // get namespace to display (either direct or from deletion order)
     if($INPUT->str('delete')){
-        $DEL = cleanID($INPUT->str('delete'));
-        $IMG = $DEL;
-        $NS  = getNS($DEL);
+    $DEL = cleanID($INPUT->str('delete'));
```

Comment:
I would expect indention here. The indentation of this file is already a bit strange with that the if's are already indented.

Reply:
Fixed by reformatting the files in `lib/exe/`